property_test.go

```diff
@@ -0,0 +1,95 @@
+package opslevel_test
+
+import (
+	"testing"
+
+	ol "github.com/opslevel/opslevel-go/v2023"
+	"github.com/rocktavious/autopilot/v2023"
+)
+
+// TODO: template variable for reused schema
+// TODO: template variable for ID's
+// TODO: template variable for prop1,prop2,prop3
+
+func TestCreatePropertyDefinition(t *testing.T) {
+	// Arrange
```

Comment:
Could use
```go
propertyDefInput := autopilot.Register[[]ol.PropertyDefinitionInput]("property_definition_input",
ol.PropertyDefinitionInput{
  Name:   "my-prop",
  Schema: "{\"$ref\":\"#/$defs/MyProp\",\"$defs\":{\"MyProp\":{\"properties\":{\"name\":{\"type\":\"string\",\"title\":\"the name\",\"description\":\"The name of a friend\",\"default\":\"alex\",\"examples\":[\"joe\",\"lucy\"]}},\"additionalProperties\":false,\"type\":\"object\",\"required\":[\"name\"]}}}",
})
```

Then `{{ template "property_definition_input" }}` can be used as the variables (second) arg in `NewTestRequest()`

and `propertyDefInput` (or whatever it's named) can be passed into `client.CreatePropertyDefinition()` below

Reply:
Example of this found in [domain_test.go](https://github.com/OpsLevel/opslevel-go/blob/main/domain_test.go#L12-L18)